circle.yml

```diff
@@ -20,7 +20,7 @@ test:
     - diff -u <(echo -n) <(go list ./compiler/natives/src/...) # All those packages should have // +build js.
     - gopherjs install -v net/http # Should build successfully (can't run tests, since only client is supported).
     - >
-      gopherjs test --minify -v --short
+      ulimit -s 10000 && ulimit -s && gopherjs test --short -v --minify
```

Comment:
I understand `ulimit -s 10000` sets maximum stack size. Wha does `ulimit -s` (the second command) do?

Reply:
Also, please don't change order of flags (unless you have a reason to be doing it). I placed `--minify` first on purpose in the previous commits to signify it's the most important flag.

My understanding this flag switch is because you rebased the entire line without changing it, is that right?

Reply:
`ulimit -s` is a means of echo-ing out the value of the OS-level stack size limit; I included it as a means of confirming the previous command was having the intended effect: not strictly necessary so I've removed it.

> Also, please don't change order of flags (unless you have a reason to be doing it). I placed --minify first on purpose in the previous commits to signify it's the most important flag.

Ok. 

> My understanding this flag switch is because you rebased the entire line without changing it, is that right?

Yes.